chatgpt-shell.el

```diff
@@ -2053,7 +2053,7 @@ Very much EXPERIMENTAL."
   (interactive)
   (unless (eq major-mode 'chatgpt-shell-mode)
     (user-error "Not in a shell"))
-  (let* ((path (read-file-name "Restore from: " nil nil t))
+  (let* ((path (read-file-name "Restore from: " (file-name-as-directory shell-maker-transcript-default-path) nil t))
```

Comment:
file-name-as-directory will cause an error out of the box as shell-maker-transcript-default-path is nil. We can likely just use shell-maker-transcript-default-path on its own here.